requirements/cuda.txt

```diff
@@ -6,8 +6,11 @@ numba == 0.61.2; python_version > '3.9'
 
 # Dependencies for NVIDIA GPUs
 ray[cgraph]>=2.43.0, !=2.44.* # Ray Compiled Graph, required for pipeline parallelism in V1.
-torch==2.6.0
-torchaudio==2.6.0
+--extra-index-url https://download.pytorch.org/whl/cu128
```

Comment:
ditto on hardcoding cu128 here

Reply:
I'll move the check to the Dockerfile to match the CUDA_VERSION there